### ISSUE TITLE
Routines trigger with offset and remaining Time is not displayed correctly

### ISSUE THREAD

**Issue by royalheadwear** [diyhue/diyHue#184]
When I add a Timer in the Hue App (for Android) 2 hours are added on top of the time that I chose in the time selector. Surprisingly the Timer is triggered at the correct time.

When I set a wake-up-routine, the light start 2 hours after the Time i set up in the Hue App.

When I searched for the source of this bug I found out that the bridge seams to save the creation time in UTC-timezone. On the other hand all other scheduled events are saved in local time (in my case its CET).

Maybe the only thing to change would be to change the _starttime_ or the _created_ time when saving this.

I used the `/api/<username>/schedules` endpoint to check what times are saved bridge internaly.

I'm running diyHue as a docker container on a raspberry pi. On the pi the correct time zone was selected.

Is there a simple Fix for these time shifts?

**Comment by mariusmotea** [comment]
Is the clock on the bridge host correct? Did you set you timezone in Hue app?

**Comment by royalheadwear** [comment]
the time on the docker hosting raspberry pi is correct. Do I need to specify the timezone for the container as well?

I've set the timezone via the *config.json* due to "Error on loading the timezone" in the hue app. It looks like this issue should have been fixed, as mentioned in #40.

**Comment by royalheadwear** [comment]
okay. I just looked inside the docker container and found out htat it acually did run on utc-timezone.

I fixed it with the following steps:
1. stop the docker container: `docker stop diyHue`
1. remove the docker container: `docker rm diyHue`
1. run the docker container with some additional volumes: `docker run -d --name "diyHue" --network="host" -v '/mnt/hue-emulator/export/':'/opt/hue-emulator/export/':'rw' -v '/etc/timezone':'/etc/timezone':'ro' -v '/etc/localtime':'/etc/localtime':'ro' diyhue/core:latest`

As far as I understood this links the time(-zone)-information from the docker host to the docker container.

Timers show the right rest time in the Hue app now and wake up routines get triggered at the correct time now. Maybe this should be added to the docs.
